calc/src/calc_fee.rs

```diff
@@ -50,7 +50,8 @@ impl Multiplier {
             ("westend", v) if 31 <= v => V2(new_u128(inner)),
 
             ("dock-main-runtime", _) => V2(new_u128(inner)),
-            ("dock-testnet", _) => V2(new_u128(inner)),
```

Comment:
Is this chain gone from the world forever?

Reply:
Yes.